<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sunrise</class>
 <widget class="QMainWindow" name="Sunrise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sunrise</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="overallProgressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>500</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>118</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>400</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #99FF00, stop: 1.0 #04B408);
border-radius: 10px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="runtimesButton">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Runtimes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="Line" name="dividerLine">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>20</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>212</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="projectsListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="projectIconLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>resources/project-icon-placeholder.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="projectNameLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>projectNameLabel</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="webEngineView" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>640</width>
      <height>240</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="url" stdset="0">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="projectsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Projects:</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectPublisherLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>projectPublisherLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectUptimeLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>projectUptimeLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="fileProgressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>500</width>
      <height>8</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>progressLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="projectWebsiteButton">
    <property name="geometry">
     <rect>
      <x>839</x>
      <y>20</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/website.svg</normaloff>resources/website.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="gearButton">
    <property name="geometry">
     <rect>
      <x>889</x>
      <y>20</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/settings-app.svg</normaloff>resources/settings-app.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <zorder>webEngineView</zorder>
   <zorder>overallProgressBar</zorder>
   <zorder>runtimesButton</zorder>
   <zorder>settingsButton</zorder>
   <zorder>dividerLine</zorder>
   <zorder>logoLabel</zorder>
   <zorder>projectsListWidget</zorder>
   <zorder>projectIconLabel</zorder>
   <zorder>projectNameLabel</zorder>
   <zorder>projectsLabel</zorder>
   <zorder>projectPublisherLabel</zorder>
   <zorder>projectUptimeLabel</zorder>
   <zorder>fileProgressBar</zorder>
   <zorder>progressLabel</zorder>
   <zorder>playButton</zorder>
   <zorder>projectWebsiteButton</zorder>
   <zorder>gearButton</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
